src/glmfit.jl

```diff
@@ -451,7 +450,10 @@ function StatsBase.fit!(m::AbstractGLM;
         Base.depwarn("'convTol' argument is deprecated, use `atol` and `rtol` instead", :fit!)
         rtol = kwargs[:convTol]
     end
-    if !issubset(keys(kwargs), (:maxIter, :minStepFac, :convTol))
+    if haskey(kwargs, :verbose)
+        Base.depwarn("""`verbose` argument is deprecated, use `ENV["JULIA_DEBUG"]=GLM` instead.""", :negbin)
```

Comment:
I assume you mean
```suggestion
        Base.depwarn("""`verbose` argument is deprecated, use `ENV["JULIA_DEBUG"]=GLM` instead.""", :fit!)
```
?

Reply:
indeed